go.mod

```diff
@@ -1,35 +1,37 @@
 module github.com/openshift/machine-api-provider-gcp
 
-go 1.22.0
+go 1.23.0
 
-toolchain go1.22.1
+toolchain go1.23.5
 
 require (
 	github.com/blang/semver v3.5.1+incompatible
 	github.com/go-logr/logr v1.4.2
-	github.com/googleapis/gax-go/v2 v2.12.0
-	github.com/onsi/ginkgo/v2 v2.20.1
-	github.com/onsi/gomega v1.34.2
-	github.com/openshift/api v0.0.0-20240919193929-2669d1ebc910
-	github.com/openshift/library-go v0.0.0-20240919205913-c96b82b3762b
-	github.com/openshift/machine-api-operator v0.2.1-0.20240924110326-1efafa4a6615
-	golang.org/x/oauth2 v0.21.0
-	google.golang.org/api v0.152.0
-	k8s.io/api v0.31.1
-	k8s.io/apimachinery v0.31.1
-	k8s.io/apiserver v0.31.1
-	k8s.io/client-go v0.31.1
-	k8s.io/component-base v0.31.1
+	github.com/googleapis/gax-go/v2 v2.13.0
+	github.com/onsi/ginkgo/v2 v2.22.1
+	github.com/onsi/gomega v1.36.2
+	github.com/openshift/api v0.0.0-20250131155403-30a036067514
+	github.com/openshift/library-go v0.0.0-20250203131244-80620876b7c2
+	github.com/openshift/machine-api-operator v0.2.1-0.20250122132849-b397e885dafb
```

Comment:
This will need to be bumped to https://github.com/openshift/machine-api-operator/pull/1325 once that merges. So this effectively depends on that one.

Reply:
@theobarberbany FYI https://github.com/openshift/machine-api-operator/pull/1325 just merged :)

Reply:
Awesome, have updated :)